fwupdate/resources/profile_schema.avsc

```diff
@@ -0,0 +1,40 @@
+{  
+    "type":"record",
+    "name":"DeviceProfile",
+    "namespace":"org.kaaproject.demo.fwupdate",
+    "fields":[
+        { "name":"serialNumber", "type":"int"},
+        { "name":"model", "type":{ "type":"string", "avro.java.string":"String"}},
+        { "name":"location", "type":{ "type":"string", "avro.java.string":"String"}},
+        { "name":"sensors", "type": {  
+                "type":"array",
+                "items":{  
+                    "type":"record",
+                    "name":"Sensor",
+                    "namespace":"org.kaaproject.demo.fwupdate",
+                    "fields":[  
+                        { "name":"type", "type": {  
+                                "type":"enum",
+                                "name":"SensorType",
+                                "namespace":"org.kaaproject.demo.fwupdate",
+                                "symbols":[ "TEMPERATURE", "HUMIDITY", "LIGHT", "PRESSURE", "ACOUSTIC"]
+                            }
+                        },
+                        { "name":"model", "type": { "type":"string", "avro.java.string":"String"}}
```

Comment:
I propose to use some JSON formatter, e.g. https://jsonformatter.curiousconcept.com/